src/components/NearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react'
+import { Box, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+
+type isMobileProps = {
+  isMobile: boolean
+}
+type objectType = {
+  id: string
+  location: string
+  picture: string
+  fName: string
+  lName: string
+}
+const NearMeProfiles = ({ isMobile }: isMobileProps) => {
+  const { classes } = useStyles()
+  const [list, setList] = useState<Array<objectType>>([])
+  const columns = isMobile ? 3 : 6
+  const mrgLeft = isMobile ? 1 : 5
+  const txtAlign = isMobile ? 'center' : 'left'
+  const getData = () => {
+    fetch('users.json', {
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+    })
+      .then(function (response) {
+        return response.json()
+      })
+      .then(function (myJson) {
+        setList(myJson)
+      })
+      .catch((e) => {
+        console.log(e.message)
+      })
+  }
+  useEffect(() => {
+    getData()
+  }, [])
+  return (
+    <Box marginLeft={mrgLeft}>
```

Comment:
are you sure using of margin will have stable behaviour? In my opinion grid is better

Reply:
Margin-left values have been removed, and the layout has been refactored to use Grid instead of Box.